<launch>
    <node name="position_crop_node" pkg="position_crop" type="position_crop_node.py" respawn="true" output="screen"/>
</launch>
